src/badguy/zeekling.cpp

```diff
@@ -182,18 +188,120 @@ Zeekling::should_we_dive() {
 
   return false;
 }
+/**
```

Comment:
I think the maths functions are better suited as part of the vector class. 

@maxteufel What do you think?

Reply:
Yes, they should be separated.